### ISSUE TITLE
[Linux] k8s api server does not start

### ISSUE THREAD

**Issue by evertonlperes** [rancher-sandbox/rancher-desktop#1108]
### Rancher Desktop Version

0.7.0-beta.1-89-gac54837

### Rancher Desktop K8s Version

1.21.7

### What operating system are you using?

Ubuntu

### Operating System / Build Version

Ubuntu 20.04.3 LTS and Leap 15.3

### What CPU architecture are you using?

x64

### Windows User Only

N/A

### Actual Behavior

RD gets stuck on `Waiting Kubernetes API...` .
Seems to be happening on others linux distro: Leap 15.3, popOS and Mint 20.

### Steps to Reproduce

1. Install RD latest Dev package (deb)
2. Start the application
3. Should see the app stuck on  `Waiting Kubernetes API...`.

### Result

![Screen Shot 2021-12-14 at 1 07 55 PM](https://user-images.githubusercontent.com/37411123/146099306-52271ee2-cbf8-4ba6-9a91-00ab5e148025.png)

### Expected Behavior

Should start the application properly.

### Additional Information

I did a couple k8s upgrade/downgrade and looks like, some version are starting the k8s api server properly, but isn't starting the node.

* Startup RD using k8s 1.21.7 --> hangs on Waiting for K8s Api
* Startup RD using k8s 1.21.5 --> hangs on Waiting for K8s Api
* Startup RD using k8s 1.20.13 --> k8s API start but hangs on Waiting for nodes

```bash
kubectl get componentstatuses
Warning: v1 ComponentStatus is deprecated in v1.19+
NAME                 STATUS    MESSAGE   ERROR
scheduler            Healthy   ok        
controller-manager   Healthy   ok   

kubectl get nodes -A
No resource found

kubectl get pods -A
NAMESPACE     NAME                                      READY   STATUS    RESTARTS   AGE
kube-system   helm-install-traefik-9cv65                0/1     Pending   0          3m6s
kube-system   local-path-provisioner-5ff76fc89d-jbtck   0/1     Pending   0          3m2s
kube-system   metrics-server-86cbb8457f-4p4cb           0/1     Pending   0          3m2s
kube-system   coredns-6488c6fcc6-btqrs                  0/1     Pending   0          3m2s
```

![Screen Shot 2021-12-14 at 5 12 33 PM](https://user-images.githubusercontent.com/37411123/146098652-b419dda5-1403-499c-9085-d64d9eeb5885.png)

**Logs**
[lima.ha.stderr.log](https://github.com/rancher-sandbox/rancher-desktop/files/7715496/lima.ha.stderr.log)
[lima.ha.stdout.log](https://github.com/rancher-sandbox/rancher-desktop/files/7715497/lima.ha.stdout.log)
[lima.log](https://github.com/rancher-sandbox/rancher-desktop/files/7715498/lima.log)
[lima.serial.log](https://github.com/rancher-sandbox/rancher-desktop/files/7715499/lima.serial.log)

**Comment by mook-as** [comment]
The end of `lima.log` is:
```
 * Caching service dependencies ... [ ok ]
 * /var/log/k3s.log: creating file
 * Starting k3s ... [ ok ]
2021-12-14T23:31:04.710Z: Could not read k3s.yaml; retrying...
```
So things looked okay, except that k3s isn't working correctly. Can you check the k3s logs? (If it failed to tail, check `/var/log/k3s.log` in the VM.)

**Comment by evertonlperes** [comment]
@mook-as, this is the content of `k3s.log`:

```bash
tail: can't open '/var/log/k3s': No such file or directory
```

**Comment by mook-as** [comment]
I managed to reproduce this; I see in `/var/log/k3s.log` (in the VM) many repeated lines of:
```
time="2021-12-15T01:24:38.138201900Z" level=warning msg="unable to get global unicast ip from interface name: route ip+net: no such network interface"
time="2021-12-15T01:24:38.138234954Z" level=fatal msg="invalid node-ip "
```
@jandubois you've been playing with lima networking recently; is anything there affecting this?

**Comment by mook-as** [comment]
This actually _was_ fixed by #1111.
